services/QuillLMS/Gemfile

```diff
@@ -72,6 +72,7 @@ gem 'configs'
 gem 'rack-test', '~> 0.6.3'
 gem 'secure_headers', '6.3.2'
 gem 'faker'
+gem 'cgi', '~> 0.3.6'
```

Comment:
Perhaps this diff goes away after rebasing?

Reply:
So, I needed to manually put in this gem here because we were getting an RSpec error on one RSpec test after upgrading the ruby version. I looked it up and it turns out that on the new ruby version 2.7.8 we need to specify a certain version of this gem, cgi. 

https://discuss.rubyonrails.org/t/invalid-domain-example-com-in-rspec-after-changing-session-store-to-domain-all/81922